Gems/SceneProcessing/Code/Source/Generation/Components/MeshOptimizer/MeshOptimizerComponent.cpp

```diff
@@ -185,7 +106,7 @@ namespace AZ::SceneGenerationComponents
         auto* serializeContext = azrtti_cast<AZ::SerializeContext*>(context);
         if (serializeContext)
         {
-            serializeContext->Class<MeshOptimizerComponent, GenerationComponent>()->Version(4);
+            serializeContext->Class<MeshOptimizerComponent, GenerationComponent>()->Version(2);
```

Comment:
Question: is this version downgrade going to cause any issues with assets that were processed before the revert?

Reply:
That is all fine, we already had to back a previous version of this earlier and were not running into any issues.